src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -120,6 +129,104 @@ public Optional<SaveCaseResult> archiveDraftAppeal(String oauth2Token, SyaCaseWr
             .build());
     }
 
+    public Long submitAppeal(SyaCaseWrapper appeal, String userToken) {
+
+        IdamTokens idamTokens = idamService.getIdamTokens();
+
+        Long caseId = appeal.getCcdCaseId() != null ? Long.valueOf(appeal.getCcdCaseId()) : null;
+
+        log.info("Converting sya appeal data to sscs case");
+
+        SscsCaseData caseData = convertAppealToSscsCaseData(appeal);
```

Comment:
can we pls reuse the logic of convertAppealToSscsCaseData depend on V1 and V2

Reply:
It is using V1 logic - https://github.com/hmcts/sscs-tribunals-case-api/pull/4091/files#diff-c4b9e1d7d23c29d97e95f586ea705f1f8e4b729e31160deff54715289e97e1eeR150